content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -141,8 +139,6 @@ allowed to be used as inline volumes within a Pod spec may do so by:
 
 ### Generic ephemeral volumes
 
-{{< feature-state for_k8s_version="v1.23" state="stable" >}}
```

Comment:
This fact is still true. Until v1.25 disappears from https://kubernetes.io/docs/home/supported-doc-versions/, we should leave it.

The shortcode tells people when something graduated; we don't link its removal to the removal of the associated feature gate.